src/mod_muc_sql.erl

```diff
@@ -236,6 +238,27 @@ set_nick(LServer, Host, From, Nick) ->
 	end,
     ejabberd_sql:sql_transaction(LServer, F).
 
+-spec decode_affiliation(Arg :: binary()) -> atom().
+decode_affiliation(<<"owner">>) -> owner;
+decode_affiliation(<<"member">>) -> member;
+decode_affiliation(<<"outcast">>) -> outcast;
+decode_affiliation(<<"muted">>) -> muted;
+decode_affiliation(_) -> none.
+
+disable_affiliation(_Host, _LUser) ->
```

Comment:
Why are you ignoring the value if you're using it below?

Reply:
In several places it looks

Reply:
I copied variable names from the unimplemented functions, which all had _ in front of it. I thought it was a convention for this module but it was just cause they weren't used :D